src/test/java/uk/gov/hmcts/darts/cases/mapper/EventMapperTest.java

```diff
@@ -30,8 +32,28 @@ void testMapGetEventsByCaseId() {
         assertEquals("eventName", event.getName());
         assertEquals("event text", event.getText());
         assertEquals(hearingDate, event.getTimestamp());
+        assertFalse(event.getIsDataAnonymised());
     }
 
+    @Test
+    void testMapGetEventsByCaseIdDataAnonymised() {
+        OffsetDateTime hearingDate = OffsetDateTime.parse("2024-07-01T12:00Z");
+        HearingEntity hearing = CommonTestDataUtil.createHearing("case1", hearingDate.toLocalDate());
+        EventEntity eventEntity = createEventWith("eventName", "event text", hearing, hearingDate);
+        eventEntity.setDataAnonymised(true);
+        List<EventEntity> eventEntityList = Lists.newArrayList(eventEntity);
+        List<Event> events = EventMapper.mapToEvents(eventEntityList);
+        Event event = events.get(0);
+        assertEquals(1, event.getId());
+        assertEquals(102, event.getHearingId());
+        assertEquals(hearingDate.toLocalDate(), event.getHearingDate());
+        assertEquals("eventName", event.getName());
+        assertEquals("event text", event.getText());
+        assertEquals(hearingDate, event.getTimestamp());
```

Comment:
minor: 
these verifications are already taken care of by another test we have on this class `testMapGetEventsByCaseId`. I think we can remove them and leave only line 53 which is the focus of this test

Reply:
I have converted these tests to use ParameterizedTest to avoid duplication